### DISCUSSION TITLE
How to create new LS-DYNA card which could be read by OpenRadioss CFG file?

### DISCUSSION THREAD

**Discussion by halfwitbm** [OpenRadioss/OpenRadioss#1687] [Q&A]
Hi everyone,

Link" https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/6094849/OpenRadioss+Reader+Radioss+Block+Format#.cfg-Files " provide a method of new Radioss card reading by CFG files.  Does anyone else has the experience in creating new ls-dyna card which could be read by OpenRadioss CFG file? Please share the experience if you have.  Thanks.

Bin

**Comment by PaulAltair** [comment]
Thank you for your interest, This isn't possible yet, there are some further parts of the code that need to opened to allow it, but is being actively worked on and will be available soon.

**Reply by halfwitbm** [reply]
Thank you very much for the answer!

Bin
